prov/rxm/src/rxm_fabric.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016 Intel Corporation, Inc.  All rights reserved.
+ * Copyright (c) 2016, 2022 Intel Corporation, Inc.  All rights reserved.
```

Comment:
This comment is for the commit message:

Please keep the subject line to about 72 characters and ensure it's stand-alone.  It must be parse-able using git log --oneline.

Reply:
Done.